scripts/plot_outputs.R

```diff
@@ -0,0 +1,45 @@
+## ===============================#
+## Setup ---------
+## ===============================#
+
+library(tidyverse)
```

Comment:
Nit: is the convention really to have a blank line after the block comments?

Reply:
According to ChatGPT, leaving a blank line is the right convention. I couldn't find it in the official style guide though -- the closest I came to was this: https://medium.com/@sdshwetadixit/comments-and-headers-in-r-a-guide-for-all-2faae3bdc65c.